app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_from, file_to = command.split()
+    if file_from == file_to:
+        return
+    with open(file_from, "r") as file_from:
+        with open(file_to, "w") as file_to:
```

Comment:
you can do it using one `with`
```suggestion
    with (open(file_from, "r") as file_in,
             open(file_to, "w") as file_out):
```